server.go

```diff
@@ -1963,38 +1965,57 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 
 // SetSendCompressor sets a compressor for outbound messages.
 // It must not be called after any event that causes headers to be sent
-// (see SetHeader for a complete list). Provided compressor is used when below
+// (see _ServerStream_.SetHeader for a complete list). Provided compressor is used when below
```

Comment:
Sorry!  I put the underscores in there to emphasize what I was recommending changing.  Please remove them.

Also, please re-wrap comments at 80 columns, thanks!